### ISSUE TITLE
Implicit declaration of function 'RCTLogWarn' is invalid in C99

### ISSUE THREAD

**Issue by carlmagumpara** [react-native-datetimepicker/datetimepicker#811]
# Bug report

<img width="1175" alt="Screen Shot 2023-09-13 at 2 16 13 PM" src="https://github.com/react-native-datetimepicker/datetimepicker/assets/14119484/c8f00a8e-bef0-4474-b82e-494e85af1f39">

## Steps to reproduce

1. npm install @react-native-community/datetimepicker --save
2. build in xCode 12

Describe what you expected to happen:

1. No error
2. …

## Environment info

`npx react-native info` output:

```bash
System:
    OS: macOS 10.15.7
    CPU: (8) x64 Intel(R) Core(TM) i5-8257U CPU @ 1.40GHz
    Memory: 55.91 MB / 8.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 14.18.2 - ~/.nvm/versions/node/v14.18.2/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v14.18.2/bin/yarn
    npm: 6.14.15 - ~/.nvm/versions/node/v14.18.2/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.11.3 - /Users/carlmagumpara/.rvm/gems/ruby-2.7.0/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.0, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
    Android SDK: Not Found
  IDEs:
    Android Studio: 4.0 AI-193.6911.18.40.6626763
    Xcode: 12.0/12A7209 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.15 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: 0.66.4 => 0.66.4 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

Time zone name (If the problem you have is related to unexpected time / date. See list in https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).

datetimepicker version: ^7.5.0

iOS / Android version: IOS 11

**Comment by Nasseratic** [comment]
@carlmagumpara That's a super old version of react native, can you reproduce it in newer versions? 😅

**Comment by becle** [comment]
react-native -v
react-native-cli: 2.0.1
react-native: 0.68.1

same error

**Comment by becle** [comment]
i donwgraded to 7.4.2 
now successfully built

**Comment by AEP20** [comment]
to add "#import <React/RCTLog.h>" on the above of the file "RNDateTimePickerShadowView" resolved my issue

**Comment by erikkjernlie** [comment]
Thanks @AEP20, that worked for me!

**Comment by DanielAraldi** [comment]
> to add "#import <React/RCTLog.h>" on the above of the file "RNDateTimePickerShadowView" resolved my issue

Thanks, that worked for me! I am using `react-native-datetimepicker` **7.6.0** with RN **0.68.2** version.

Note: I add the import in the top file  `RNDateTimePickerShadowView.h`:
```h
#import <React/RCTShadowView.h>
#import <React/RCTLog.h>  // <--
#import "RNDateTimePicker.h"
```

**Comment by vonovak** [comment]
:tada: This issue has been resolved in version 7.6.2 :tada:

If this package helps you, consider [sponsoring us](https://github.com/sponsors/react-native-datetimepicker)! :rocket:

**Comment by shahanaahmad** [comment]
> 🎉 This issue has been resolved in version 7.6.2 🎉
> 
> If this package helps you, consider [sponsoring us](https://github.com/sponsors/react-native-datetimepicker)! 🚀

This worked for me. Thanks!
